api/lib/domain/constants/update-organization-learners-password-errors.js

```diff
@@ -0,0 +1,10 @@
+const ORGANIZATION_LEARNER_WITHOUT_USERNAME_CODE = 'ORGANIZATION_LEARNER_WITHOUT_USERNAME';
+const USER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE = 'USER_DOES_NOT_BELONG_TO_ORGANIZATION';
+const ORGANIZATION_LEARNER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE =
+  'ORGANIZATION_LEARNER_DOES_NOT_BELONG_TO_ORGANIZATION';
+
+export {
+  ORGANIZATION_LEARNER_WITHOUT_USERNAME_CODE,
+  ORGANIZATION_LEARNER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE,
+  USER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE,
+};
```

Comment:
Mettre un ordre plus logique pour les codes d'erreur en mettant l'ordre dans lequel les erreurs peuvent arriver : 
```suggestion
const USER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE = 'USER_DOES_NOT_BELONG_TO_ORGANIZATION';
const ORGANIZATION_LEARNER_WITHOUT_USERNAME_CODE = 'ORGANIZATION_LEARNER_WITHOUT_USERNAME';
const ORGANIZATION_LEARNER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE =
  'ORGANIZATION_LEARNER_DOES_NOT_BELONG_TO_ORGANIZATION';

export {
  USER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE,
  ORGANIZATION_LEARNER_WITHOUT_USERNAME_CODE,
  ORGANIZATION_LEARNER_DOES_NOT_BELONG_TO_ORGANIZATION_CODE,
};
```